portal-api/src/v1/controllers/feedback.controller.js

```diff
@@ -38,18 +37,14 @@ exports.create = async (req, res) => {
 
   const modifiedFeedback = {
     ...req.body,
-    submittedBy: req.user.username,
-    created: now(),
+    created: new Date(),
```

Comment:
and can we include the `submittedBy` property as well? this should be populated only if the user is logged in.

Reply:
since I changed to openRouter and removed validation, user does not show in the request anymore

Reply:
this should be present in the request if the user is logged in. we can add a check for this in the code.

Reply:
sonw